config/crd/bases/approuting.kubernetes.azure.com_clusterexternaldnses.yaml

```diff
@@ -51,6 +51,8 @@ spec:
                   or private). The number of zones is currently capped at 7 but may
                   be expanded in the future.
                 items:
+                  maxLength: 172
```

Comment:
I believe this is unintentional? not sure we should be seeing any changes here

Reply:
the min length is odd too

Reply:
Going through spec - that's actually expected

https://github.com/Azure/aks-app-routing-operator/blob/2388d8c57bda42f7ef58e4f5dc4bb4b375b844db/api/v1alpha1/clusterexternaldns_types.go#L59

Again, just not sure why kubebuilder is being inconsistent with output

Reply:
@meecethereese would you mind adjusting the spec on externaldns CRDs to remove the min/max len fields on dns zone resource ids? We already have validation within the manifests, and CEL validation was too complex to ensure each item actually resembled an Azure resource ID